### PR TITLE
Basic Layout Setup & First Send Page

### PR DESCRIPTION
**Pull request by 0xLe0x**

### CODE REVIEW

File: packages/app/components/layout/footer/components/SendButton.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Button, Image, YStack } from "@my/ui"
+import SendLogo from "app/assets/img/logos/send-app-logo.svg"
```
Comment (0xBigBoss) [inline packages/app/components/layout/footer/components/SendButton.tsx:2]: why on this one do we import as a SVG file?

File: packages/app/features/send/components/numpad/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { XStack, YStack } from "@my/ui"
+import { NumPadButton } from "./numpad-button"
+
+interface INumPadProps {
+  value: string
+  setValue: (val: string) => void
+}
+
+export const NumPad = ({ value, setValue }: INumPadProps) => {
+  const numpadpressHandler = (val: string) => {
```
Comment (0xBigBoss) [inline packages/app/features/send/components/numpad/index.tsx:10]: i would move to utils to instead return this value and add some unit tests using vitest like so 

https://github.com/0xsend/sendapp/blob/473ece3c6459a661c0ecd24fb14c4b7ddf302d35/packages/app/utils/formatAmount.test.ts#L1

File: packages/app/features/send/components/numpad/numpad-button.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { Button, ButtonProps, styled } from "@my/ui"
+
+const CustomButton = styled(Button, {
```
Comment (0xBigBoss) [inline packages/app/features/send/components/numpad/numpad-button.tsx:3]: rename NumpadButton

File: packages/app/features/send/components/select/select.tsx
Comment (0xBigBoss) [inline packages/app/features/send/components/select/select.tsx:1]: ❤️

File: packages/app/features/send/components/select/select.tsx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+import { IconTriangleDown } from "app/components/icons/IconTriangleDown"
+import { useMemo, useState } from "react"
+import { IconProps } from "@tamagui/helpers-icon"
+
+export function Select({ items, ...props }: SelectProps & { items: Array<{ icon?: React.ReactNode, name: string }> }) {
+  const [val, setVal] = useState(items[0]?.name.toLowerCase())
+
+  return (
+    <TamaguiSelect
+      id="food"
```
Comment (0xBigBoss) [inline packages/app/features/send/components/select/select.tsx:13]: i would remove

**Review by 0xBigBoss — APPROVED** [review]
There's a bunch of places where I would leave TODO's to come back and clean it up. just a few questions and little details.

thx @0xLe0x

File: packages/app/features/send/components/numpad/index.tsx (hunk 2 above)
Comment (0xLe0x) [reply packages/app/features/send/components/numpad/index.tsx]: added `formatNumpadInput.ts` & `formatNumpadInput.test.ts`

**Pull request merged**